api/models/config_keys.py

```diff
@@ -81,6 +81,17 @@ class CoreConfig(SnakeModel):
     QueueAuthKey: Optional[str] = Field(None, example='shibalba')
 
 
+class AreaConfigDTO(BaseModel):
+    name: str = Field(example='Kitchen')
+    id: str = Field(example='area0')
+    cameras: Optional[str] = Field("", example='cam0,cam1')
+    occupancyThreshold:  Optional[int] = Field(0, example=300)
+    occupancyNotificationsEmails: Optional[str] = Field("", example='john@email.com,doe@email.com')
+    notifyEveryMinutes: Optional[int] = Field(0, example=15)
+    violationThreshold: Optional[int] = Field(0, example=100)
+    violationsNotificationsEmails: Optional[str] = Field("", example='john@email.com,doe@email.com')
+
+
```

Comment:
I think that we can refactor a little the class and avoid repeating some code.

What do you think about something like?
```
class EntityConfigDTO(BaseModel):
    name: str = Field(example='Kitchen')
    id: str = Field(example='area0')
class NotificationConfig(BaseModel):
    violationThreshold: Optional[int] = Field(0, example=100)
    notifyEveryMinutes: Optional[int] = Field(0, example=15)
    emails: Optional[str] = Field("", example='john@email.com,doe@email.com')
    dailyReport: Optional[bool] = Field(False, example=True)
class AreaNotificationConfig(NotificationConfig):
    occupancyThreshold:  Optional[int] = Field(0, example=300)
class AreaConfigDTO(EntityConfigDTO, AreaNotificationConfig):
    cameras: Optional[str] = Field("", example='cam0,cam1')
class SourceConfigDTO(EntityConfigDTO, NotificationConfig):
    videoPath: str = Field(example='/repo/data/softbio_vid.mp4')
    tags: Optional[str] = Field("", example='kitchen,living_room')
    image: Optional[str] = Field("", example='Base64 image')
    distMethod: Optional[str] = Field("", example='CenterPointsDistance')
```